GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs

```diff
@@ -127,6 +127,38 @@ private static void UpdateGuiFocus(ToolBarSelectedButton currentTab)
             }
         }
 
+        private void OnGuiOverviewSelect()
+        {
+            var descriptionTextStyle = CreateDescriptionTextStyle();
+            EditorGUILayout.LabelField("Building Instant Game with Quick Deploy", EditorStyles.boldLabel);
+            EditorGUILayout.BeginVertical(UserInputGuiStyle);
+            EditorGUILayout.Space();
+            EditorGUILayout.Space();
+            EditorGUILayout.BeginHorizontal();
+            EditorGUILayout.LabelField("Building an instant game with Quick Deploy will significantly reduce the size" +
+                                       " of your game by storing the main AssetBundle of the game to a remote server.",
+                descriptionTextStyle);
+            EditorGUILayout.EndHorizontal();
+            EditorGUILayout.Space();
+            EditorGUILayout.Space();
+            EditorGUILayout.BeginHorizontal();
+            EditorGUILayout.LabelField("Use Quick Deploy features to Build and Deploy AssetBundle, as well as to " +
```

Comment:
`"Use the \"Bundle Creation\" tab to build an AssetBundle containing the game's main scene. Then upload the AssetBundle to Google Cloud Storage via the \"Bundle Deployment\" tab. Finally, use the \"Loading Screen\" tab to select an image to display on the loading screen and the URL that points to the uploaded AssetBundle."`

Reply:
Resolved.